libs/react/shelter/src/lib/pages/shelter/GeneralInfo/index.tsx

```diff
@@ -100,7 +100,7 @@ export default function GeneralInfo({
           <>
             <h3 className="text-base font-semibold">Bed Availability</h3>
             <div className="flex items-center justify-between gap-1">
-              <p>{shelter.totalBeds} beds available</p>
+              <p>Capacity: {shelter.totalBeds} beds</p>
```

Comment:
I wonder if line 101 should also be updated or removed. Not clear in the ticket, so this lgtm